Documentation/Maintenance/Release.md

```diff
@@ -551,26 +551,19 @@ Create a PR to update
 the new version. This conda recipe downloads the wheel binary packages and
 re-packages them as conda packages.
 
+Note: A `post1` wheel corresponds to bumping the *build.number* field in
+*recipe/meta.yml*. `POST` variables in *bld.bat* and *build.sh* are available
+to specify an options wheel post version.
```

Comment:
to specify an options' wheel post version?

Reply:
Good catch -- should be "optional".

Reply:
Done in ea85dfe.